src/components/core/DpEditor/DpEditor.stories.mdx

```diff
@@ -16,4 +16,7 @@ A rich text editor component based on tiptap.
 - the editor-content component needs a prop with editor instance to work correctly
 
 To properly set the content we have to update this.currentValue and use editor.setContent() - both actions are needed!
-To use boilerplates we need to mount tiptap editor, where boilerplate modal is imported dynamically. The boilerplates are stored in boilerplates vuex store, which is also registered dynamically if boilerplate prop is specified.
+
+To use boilerplates we need to mount tiptap editor, where boilerplate modal is imported dynamically.
+
+The boilerplates are stored in boilerplates vuex store, which is also registered dynamically if boilerplate prop is specified.
```

Comment:
this is dplan specific. So it should not be mentioned here.
But maybe we can add a configuration where the slots are used. Something like the imageUpload modal ?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/df50c169fd1b50ab220d9f778e973311f2f03123